packages/core/src/components/Cards/AnimatedPictureCard/AnimatedPictureCard.js

```diff
@@ -1,16 +1,22 @@
-import PropTypes from 'prop-types';
+//@flow
```

Comment:
It's nice having Flow, but I think ESLint would complain about not having the space in between `//` and `@flow` 🤣